### PR TITLE
Introduced the concept of view events

### PR DESCRIPTION
**Pull request by Kazark**
I find this to be superior to a bunch of subscribe methods on the view
interface. What do you think, Gilli?

### CODE REVIEW

**Comment by Kazark** [comment]
I thought I should introduce this before we started to try out other GUI frameworks so we had our interface somewhat standardized (until it evolves further... :).

**Comment by Kazark** [comment]
@mjgpy3 No pressure, but if you haven't had a chance to merge this by the time I am ready to submit a pull request for the WinForms stuff, I'll merge this. Now you probably have a week or two left yet before I'll be ready for that. Just a heads up. It looks like it's going to fix all our current major issues though so I'm not thinking there's a need to explore other GUIs at this time.

File: Void/Void.ViewModel/VoidController.fs [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 open Void.Core
 open CellGrid
+open System
```
Comment (mjgpy3) [inline Void/Void.ViewModel/VoidController.fs]: What does this do?

Comment (mjgpy3) [inline Void/Void.ViewModel/VoidController.fs:5]: What does this do?

Nevermind, thanks Google

**Comment by Kazark** [comment]
BTW I forgot to mention the reason that I said the above: the WinForms stuff is built on the model introduced here (though it could be updated to not depend on it if you think I've gone the wrong direction here... it's just that it wouldn't make sense to have two pull requests open, one of which was dependent on the other.)

**Pull request merged**